fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanSummaryWrapper.java

```diff
@@ -247,7 +265,8 @@ public Money calculateTotalChargesRepaidAtDisbursement(Set<LoanCharge> charges,
             return total;
         }
         for (final LoanCharge loanCharge : charges) {
-            if (!loanCharge.isPenaltyCharge() && loanCharge.getAmountPaid(currency).isGreaterThanZero()) {
+            if (!loanCharge.isPenaltyCharge() && loanCharge.getAmountPaid(currency).isGreaterThanZero()
+                    && loanCharge.isDisbursementCharge()) {
```

Comment:
Was it a bug?

Reply:
Originally we were using this method to collect all paid charges. With the credit allocation the respective charge is not created, so we have to get this info from the LoanRepaymentSchedule, however that one does not contain the charges created at disbursemenet time, so I had to fix it. 

obviously the original name of this method was wrong, but in the context it was being used it was correct. Now the method name is also ok.